.luacov

```diff
@@ -1,33 +1,29 @@
 -- Configuration file for LuaCov
 
 -- This variable is set externally
-local build = os.getenv("BUILD_DIRECTORY") or error("$BUILD_DIRECTORY not set")
-local lib_dir = os.getenv("AWESOME_LIB_DIR") or build .. "lib/"
-local source = os.getenv("SOURCE_DIRECTORY")
+local source = os.getenv("SOURCE_DIRECTORY") or os.getenv('PWD')
+local build = os.getenv("BUILD_DIRECTORY") or (source .. "/build")
+local lib_dir = os.getenv("AWESOME_LIB_DIR") or (source .. "/lib")
```

Comment:
Yeah, I ended up squashing everything.
And there are reverts for previous commits (from this PR) in there..
I could look into cleaning that up again.

`set -v` prints the executed commands, but less verbose than `-x`. Travis seems to do that already / and/or it annoyed me.